envsec/envsec.go

```diff
@@ -86,15 +87,25 @@ func (c *SSMConfig) IsEnvStoreConfig() bool {
 	return true
 }
 
-func (c *SSMConfig) VarPath(envId EnvId, varName string) string {
+func (c *SSMConfig) varPath(envId EnvId, varName string) string {
```

Comment:
`varPath` is the SSM param store path for variables. (Maybe we should rename to `paramStorePath`). It used to be fixed but we allow customization so that launchpad and other 3rd parties can set what they prefer.